File: 03-02-2025/grade of students.c
#include <stdio.h>

int main() {
    float marks;
    char grade;

    // Input marks from the user
    printf("Enter marks (0-100): ");
    scanf("%f", &marks);

    // Calculate grade
    if (marks >= 90) {
        grade = 'A';
    } else if (marks >= 80) {
        grade = 'B';
    } else if (marks >= 70) {
        grade = 'C';
    } else if (marks >= 60) {
        grade = 'D';
    } else {
        grade = 'F';
    }

    // Output the grade
    printf("Grade: %c\n", grade);
    
    return 0;
}

File: 03.02.2025/grade of students.py
# Function to calculate grade based on marks
def calculate_grade(marks):
    if marks >= 90:
        return 'A'
    elif marks >= 80:
        return 'B'
    elif marks >= 70:
        return 'C'
    elif marks >= 60:
        return 'D'
    else:
        return 'F'

# Input marks from user
marks = float(input("Enter marks (0-100): "))

# Calculate and output grade
grade = calculate_grade(marks)
print("Grade:", grade)
